test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
Looks like the tests are failing because you are asserting a `Vector()==List()`. @jyundt

```
Expected: LshwRepresentation(Vector(Cpu(8,16....
Actual:      LshwRepresentation(List     (Cpu(8,16...
```

I think if you sort that out (and ensure that `equals()` on `Cpu` and `Gpu` classes work with value equality and not object identity), it should work!